### PR TITLE
[1.3.7-prepare][Fix-5467][UI] Fix UI cannot be displayed normally in some browsers

### LINKED ISSUE #4335: [Bug][UI] IE 9-11 not supported
**For better global communication, Please describe it in English. If you feel the description in English is not clear, then you can append description in Chinese(just for Mandarin(CN)), thx! **
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior, for example:
1. Go to 'http://106.75.43.194:8888/dolphinscheduler/ui/view/login/index.html' in IE 9-11
2. Go to 'http://localhost:8888' in IE 9-11
3. Cannot display properly, and see error in console

**Expected behavior**
Bug fixed and IE 9-11 supported

**Screenshots**
image
image

**Which version of Dolphin Scheduler:**
 -[all version]

**Additional context**

**Requirement or improvement**
- Please describe about your requirements or improvement suggestions.

### LINKED ISSUE #4338: [Bug][UI] Invalid date problem in IE
**For better global communication, Please describe it in English. If you feel the description in English is not clear, then you can append description in Chinese(just for Mandarin(CN)), thx! **
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior, for example:
1. Go to 'Task Instance', 'Warnning group manage' or 'Token manage' in IE
2. See 'Invalid Date' error

**Expected behavior**
Bug fixed.

**Screenshots**
image
image
image

**Which version of Dolphin Scheduler:**
 -[all versions]

**Additional context**

**Requirement or improvement**
- Please describe about your requirements or improvement suggestions.

### LINKED ISSUE #5467: [Bug][UI] UI cannot be displayed normally in some browsers
Related issue: #5457 #4335 #4338

**Screenshots**

Browsers:

- IE 10, IE 11
- 360 browser version 12.2.1172.0 chromium core version 78.0.3904.108

image
image
image
image

**Expected behavior**
Bug fixed

The following browsers hope to be supported

```
"> 1%",
"last 2 versions",
"ie >= 10",
"edge >= 12",
"firefox >= 28",
"chrome >= 29",
"safari >= 10",
"opera >= 17"
```

The following browsers may need to be tested

```
"QQ Browser >= 10.4"
"Baidu Browser >= 7.12"
"360 Browser"
"Sogou Browser"
```

**Which version of Dolphin Scheduler:**
 -[1.3.x]

### PR DESCRIPTION
**Pull request by chengshiwen**
pr: #4336 #4339
issue: #5467 #4335 #4338

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/index.js [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
+import 'babel-polyfill'
 import Vue from 'vue'
 import App from './App'
 import router from './router'
```
Comment (wen-hemin) [inline dolphinscheduler-ui/src/js/conf/home/index.js:23]: Keep spaces

File: dolphinscheduler-ui/src/js/conf/login/index.js [hunk 2]
```diff
@@ -18,6 +18,7 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
 // import $ from 'jquery'
+import 'babel-polyfill'
 import Vue from 'vue'
 import App from './App'
 import i18n from '@/module/i18n'
```
Comment (wen-hemin) [inline dolphinscheduler-ui/src/js/conf/login/index.js:24]: Keep spaces

**Review by wen-hemin — COMMENTED** [review]
Please confirm

File: dolphinscheduler-ui/src/js/conf/home/index.js (hunk 1 above)
Comment (chengshiwen) [reply dolphinscheduler-ui/src/js/conf/home/index.js:23]: It's just ok, I don't know why it turns red

File: dolphinscheduler-ui/src/js/conf/login/index.js (hunk 2 above)
Comment (chengshiwen) [reply dolphinscheduler-ui/src/js/conf/login/index.js:24]: It's just ok, I don't know why it turns red

**Review by wen-hemin — APPROVED** [review]
+1

**Pull request merged**